iceoryx_utils/test/moduletests/test_cxx_type_traits.cpp

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) 2020 by Robert Bosch GmbH. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "iceoryx_utils/cxx/type_traits.hpp"
+#include "test.hpp"
+
+using namespace ::testing;
+using namespace iox::cxx;
+
+TEST(TypeTraitsTest, IsInvocableResolvesToTrue)
+{
+    auto lambda = [](int foo) -> void { foo++; };
+    auto sut = is_invocable<decltype(lambda), int>::value;
+    EXPECT_TRUE(sut);
+}
+
+TEST(TypeTraitsTest, IsInvocableResolvesToFalse)
+{
+    int beeblebrox{42};
```

Comment:
Long live Zaphod our president of the universe !

Reply:
isn't this a magic number? shouldn't it be like this
```
constexpr int ANSWER {42};
int beeblebrox{ANSWER};
```